<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>battleWindowForm</class>
 <widget class="QWidget" name="battleWindowForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>723</width>
    <height>273</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout">
   <item row="0" column="0">
    <layout class="QGridLayout">
     <item row="0" column="0">
      <widget class="QGraphicsView" name="mapGraphicsView"/>
     </item>
     <item row="0" column="1">
      <widget class="QTabWidget" name="optionTabs">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="gameOptionsTab">
        <attribute name="title">
         <string>Game &amp;Options</string>
        </attribute>
        <layout class="QGridLayout">
         <item row="0" column="0">
          <layout class="QVBoxLayout">
           <item>
            <widget class="QLabel" name="startPositionLabel">
             <property name="text">
              <string>Start &amp;position:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="buddy">
              <cstring>startPositionComboBox</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="gameEndLabel">
             <property name="text">
              <string>Game &amp;end:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="buddy">
              <cstring>gameEndComboBox</cstring>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="1">
          <layout class="QVBoxLayout">
           <item>
            <widget class="QComboBox" name="startPositionComboBox">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="currentIndex">
              <number>2</number>
             </property>
             <item>
              <property name="text">
               <string>Fixed</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Random</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Choose in game</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="gameEndComboBox">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="currentIndex">
              <number>0</number>
             </property>
             <item>
              <property name="text">
               <string>When last unit dies</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>When commander dies</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Lineage mode</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0" colspan="2">
          <layout class="QGridLayout">
           <item row="0" column="0">
            <widget class="QCheckBox" name="limitDGunCheckBox">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="font">
              <font>
               <pointsize>8</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Limit &amp;D-Gun to start</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QCheckBox" name="diminishingMetalMakersCheckBox">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="font">
              <font>
               <pointsize>8</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Diminishing &amp;metal makers</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QCheckBox" name="ghostedBuildingsCheckBox">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="font">
              <font>
               <pointsize>8</pointsize>
              </font>
             </property>
             <property name="text">
              <string>&amp;Ghosted buildings</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QCheckBox" name="undeformableMapameSpeedCheckBox">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="font">
              <font>
               <pointsize>8</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Undeformable map</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="1">
          <spacer>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>16</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Mod options</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QTextBrowser" name="modOptions"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="unitsTab">
        <attribute name="title">
         <string>&amp;Units</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <layout class="QHBoxLayout">
           <item>
            <widget class="QLabel" name="disabledUnitsLabel">
             <property name="text">
              <string>List of disabled Units:</string>
             </property>
             <property name="buddy">
              <cstring>restrictedUnitsListView</cstring>
             </property>
            </widget>
           </item>
           <item>
            <spacer>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>68</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="addUnitPushButton">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../resources.qrc">
               <normaloff>:/icons/trolltech/plus.png</normaloff>:/icons/trolltech/plus.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="removeUnitPushButton">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../resources.qrc">
               <normaloff>:/icons/trolltech/minus.png</normaloff>:/icons/trolltech/minus.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QListView" name="restrictedUnitsListView"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="advancedTab">
        <attribute name="title">
         <string>&amp;Advanced</string>
        </attribute>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout">
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>Fix colors</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Fix IDs</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Lock IDs &amp; teams</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ring unready players</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Balance teams</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Balance settings...</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Add Bot</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&amp;Fix &amp;&amp; Balance</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="lockGameCheckBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Loc&amp;k game</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="factionsComboBox"/>
     </item>
     <item>
      <widget class="QCheckBox" name="specCheckBox">
       <property name="text">
        <string>Spectator</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="readyCheckBox">
       <property name="text">
        <string>&amp;Ready</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="gameHostingStartPushButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Start the Game [Ctrl+Enter]</string>
       </property>
       <property name="text">
        <string>&amp;Start</string>
       </property>
       <property name="shortcut">
        <string>Ctrl+Return</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
